pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -411,6 +412,19 @@ pub mod pallet {
 			(random_tier, random_variation)
 		}
 
+		#[inline]
+		fn calculate_soul_points_from_dna(dna: &Dna, season: &SeasonOf<T>) -> SoulPoints {
```

Comment:
I would pass just `max_variations` instead of `season`.

Reply:
I think it's more explicit like this, you don't want people passing wrong values by mistake, so forcing you to pass the reference to a season is more self-documenting.